Shake/Main.lean

```diff
@@ -293,7 +293,7 @@ def visitModule (s : State) (srcSearchPath : SearchPath) (ignoreImps : Bitset)
     try
       let (path, inputCtx, header, endHeader) ← parseHeader srcSearchPath s.modNames[i]!
       for stx in header.raw[2].getArgs do
-        if toRemove.any fun i => s.modNames[i]! == stx[2].getId then
+        if toRemove.any fun i => s.modNames[i]! == stx[3].getId then
```

Comment:
Can we pattern match here so that we aren't at the whim of the underlying syntax?

Reply:
added a self-test for this purpose